timemachine/fe/single_topology.py

```diff
@@ -1123,6 +1065,27 @@ def __init__(self, mol_a, mol_b, core):
         self.c_to_a = {v: k for k, v in enumerate(self.a_to_c)}
         self.c_to_b = {v: k for k, v in enumerate(self.b_to_c)}
 
+    def get_dummy_atoms_a(self):
+        dummies = set()
+        for idx, flag in enumerate(self.c_flags):
+            if flag == AtomMapFlags.MOL_A:
+                dummies.add(idx)
+        return dummies
```

Comment:
nit/style preference: these can be one-liners with set comprehensions
```suggestion
        return {idx for idx, flag in enumerate(self.c_flags) if flag == AtomMapFlags.MOL_A}
```

Reply:
willfix